scripts/build/node_install.sh

```diff
@@ -1,14 +1,23 @@
+#!/bin/bash
 # Install the latest node version using nvm (Node Version Manager)
 timer_start=$(date +%s)
 
 echo "Install Node"
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
-export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
-nvm install --lts
-
-echo "npm version:"
-echo `npm -v`
+#comment this below line in local if nvm already installed
+export NVM_DIR=/root/.nvm
```

Comment:
This seems like something specific to only the Docker build environment. Could we move `export NVM_DIR=/root/.nvm` to the Dockerfile?

Reply:
we are using NVM_DIR in line 9-12